.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Build on Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        uses: actions/checkout@v2
+        name: Set up QEMU
+      -
+        uses: docker/setup-qemu-action@v1
+        name: Set up Docker Buildx
+      -
+        uses: docker/setup-buildx-action@v1
+        name: Login to DockerHub
+      -
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_TOKEN }}
+      -
+        name: Build and push to DockerHub
+        uses: docker/build-push-action@v2
+        with:
+          context: ./
+          file: ./Dockerfile
+          platforms: linux/amd64
+          push: true
+          tags: |
+            mpuckett259/signal-scanner-bo:latest
```

Comment:
Can we use the SDSA org for this? Also, should it be `bot` at the end instead of `bo`?